delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -745,10 +599,9 @@ def cleanupCloudFrontDistribution(service, domain, stage) {
 			//events.sendCompletedEvent('DELETE_CLOUDFRONT', null, null, getEnvKey(stage))
 		} catch(ex) {
 			if((ex.getMessage()).indexOf("getDistributionId Failed") > -1) {
-				echo "Could not find a CloudFront distribution Id for service: $service and environment $stage"
+				echo "Could not find a CloudFront distribution Id for service: ${service_config['service']} and environment $stage"
 				//events.sendCompletedEvent('DELETE_CLOUDFRONT', "CF resource not available", null, getEnvKey(stage))
 			} else {
-				updateServiceInDB(var_db_service_id, configLoader.AWS.API.PROD_ID, configLoader.AWS.REGION, "deletion_failed");
```

Comment:
send an event for failure

Reply:
Modified the code for sending failed event